src/backend/distributed/deparser/deparse_type_stmts.c

```diff
@@ -0,0 +1,427 @@
+/*-------------------------------------------------------------------------
+ *
+ * deparse_type_stmts.c
+ *	  All routines to deparse type statements.
+ *	  This file contains all entry points specific for type statement deparsing as well as
+ *	  functions that are currently only used for deparsing of the type statements.
+ *
+ *	  Functions that could move later are appendColumnDef, appendColumnDefList, etc. These
+ *	  should be reused across multimple statements and should live in their own deparse
+ *	  file.
+ *
+ * Copyright (c) 2019, Citus Data, Inc.
+ *
+ * $Id$
```

Comment:
`$Id$`?

Reply:
probably a left over of a copy somewhere